{
    "id": "CVE-2016-3963",
    "cisaKEV": false,
    "reported_exploited": false,
    "exploit_maturity": "poc",
    "counts": {
        "public_exploit_count": 2
    },
    "timeline": {
        "nvd_published": "2016-04-08"
    },
    "exploits": [
        {
            "url": "https://gitlab.com/exploit-database/exploitdb/-/blob/main/exploits/linux/dos/44721.py",
            "name": "Siemens SCALANCE S613 - Remote Denial of Service",
            "source": "exploitdb",
            "date_added": "2018-05-23",
            "exploit_maturity": "poc"
        },
        {
            "url": "https://packetstormsecurity.com/files/download/147853/siemensscalances613-dos.txt",
            "name": "Siemens SCALANCE S613 Denial Of Service",
            "source": "packetstorm",
            "date_added": "2018-05-24",
            "exploit_maturity": "poc"
        }
    ]
}